starter/source/materials/mat/mat069/law69_upd.F

```diff
@@ -160,9 +160,15 @@ SUBROUTINE LAW69_UPD(IOUT,TITR    ,MAT_ID,UPARAM,NFUNC,
        ENDIF  
 !       
        IF (IFLAG == 1 .OR. IFLAG == 2) THEN
-          CALL LAW69_NLSQF(IFLAG,STRETCH,STRESS,10,NMUAL,NOGD,MUAL,
+           LAWID = IFLAG 
```

Comment:
Hi @SaidMamouri 
can you correct indentation , to have homogeneous code ?

Thanks